calyx/frontend/src/parser.rs

```diff
@@ -26,6 +27,14 @@ use std::{fs, path::PathBuf};
 type ParseResult<T> = Result<T, Error<Rule>>;
 type ComponentDef = ast::ComponentDef;
 
+/// The connections type which holds `Wire`, `Group`, `StaticGroup`, and `FSM`.
+type ConnectionTuple = (
```

Comment:
Let's call this `ParsedConnections`. It's already self-evident that it's a tuple, so there's no need to include the word "tuple" in the name.

For the documentation, let's describe in more detail what this type is for: this holds all the results from parsing the `wires` section of a Calyx program.